openslides/motions/views.py

```diff
@@ -109,7 +109,7 @@ def create(self, request, *args, **kwargs):
         if (not request.user.has_perm('motions.can_see_and_manage_comments') and
                 request.data.get('comments')):
             # Some users are not allowed to send comments data.
-            self.permission_denied(request)
+            del request.data['comments']
```

Comment:
Why this? I think we should hide the field in the form (See MotionComment.getFormFields in site.js I think). Then the user should not submit comments if he is not authorized.